docs/sources/installation/requirements.md

```diff
@@ -7,23 +7,23 @@ type = "docs"
 name = "Requirements"
 identifier = "requirements"
 parent = "installation"
-weight = -1
+weight = 100
 +++
 
 # Requirements
 
 This page includes useful information on the supported Operating Systems as well as the hardware requirements that are needed to install and use Grafana.
 
-## Operating Systems
+## Supported operating systems
 
-### Supported
+The following operating systems are supported for Grafana installation:
 
 - [Debian / Ubuntu](/installation/debian)
 - [RPM-based Linux (CentOS, Fedora, OpenSuse, RedHat)](/installation/rpm)
 - [Mac OS X](/installation/mac)
 - [Windows](/installation/windows)
 
-### Unsupported
+## Unsupported
```

Comment:
This needs to be `Unsupported operating systems` since you've moved this to be a H2 instead of H3 - earlier the unsupported was a sub-header of `Operating systems`.

Reply:
Excellent, thanks for clarifying that!